examples/topology.example.yaml

```diff
@@ -11,10 +11,12 @@ global:
   # # See: https://www.freedesktop.org/software/systemd/man/systemd.resource-control.html
   # # Supports using instance-level `resource_control` to override global `resource_control`.
   # resource_control:
-  #   # See: https://www.freedesktop.org/software/systemd/man/systemd.resource-control.html#IOReadBandwidthMax=device%20bytes
+  #   # See: https://www.freedesktop.org/software/systemd/man/systemd.resource-control.html#MemoryLimit=bytes
   #   memory_limit: "2G"
-  #   # See: https://www.freedesktop.org/software/systemd/man/systemd.resource-control.html#IOReadBandwidthMax=device%20bytes
-  #   cpu_quota: "20%"
+  #   # See: https://www.freedesktop.org/software/systemd/man/systemd.resource-control.html#CPUQuota=
+  #   # The percentage specifies how much CPU time the unit shall get at maximum, relative to the total CPU time available on one CPU. Use values > 100% for allotting CPU time on more than one CPU.
+  #   # Example: CPUQuota=20% ensures that the executed processes will never get more than 20% CPU time on one CPU.
```

Comment:
This comment is inconsistent with the next line. Oops, all the `see: xxxx` comments are wrong.

Reply:
https://github.com/pingcap/tiup/pull/668/commits/2a9dc6a145ca2f2d97c95c2672e7474c2f5a6ab2